APIs/openEO/fair.qmd

```diff
@@ -9,23 +9,27 @@ aliases:
 
 ## FAIR
 
-One of the key goals of openEO, is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. The implementation in the Copernicus dataspace
-makes it easier to comply with these principles, by incorporating these principles in the implementation, so that
-users are automatically a step closer to generating FAIR-compliant open data. If your project has requirements related to these topics,
-this should serve as a good starting point.
+One of the goals of openEO is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. These principles are seamlessly integrated into the Copernicus Data Space Ecosystem, making it intuitive to adhere to them.
+Consequently, using openEO allows users to develop FAIR-compliant open solutions automatically. For projects with requirements in these areas, openEO offers an excellent starting point.
```

Comment:
not sure if "For projects with requirements in these areas, openEO offers an excellent starting point." is even required